gordon.toml.example

```diff
@@ -1,6 +1,10 @@
 # Gordon Core Config
-[logging]
+[core]
```

Comment:
I felt like this made more sense to separate out what could be a plugin versus core to the Gordon service.